test/service/test_response.py

```diff
@@ -3873,3 +3818,539 @@ def test_file_urls(self):
             file = self.get_file('c343a47d-683f-571d-99c4-1331841b4e63')
             self.assertIsNone(file['url'])
             self.assertIsNone(file['drs_uri'])
+
+    def test_term_facets(self):
+        """
+        Verify the term facets
+        """
+        expected = {
+            'accessible': [
+                {'term': 'true', 'count': 2}
+            ],
+            'assayType': [
+                {'term': None, 'count': 2}
+            ],
+            'biologicalSex': [
+                {'term': 'unknown', 'count': 1},
+                {'term': None, 'count': 1}
+            ],
+            'bionetworkName': [
+                {'term': None, 'count': 2}
+            ],
+            'cellLineType': [
+                {'term': 'primary', 'count': 1},
+                {'term': None, 'count': 1}
+            ],
+            'contactName': [
+                {'term': 'Chanjuan,,Shen', 'count': 1},
+                {'term': 'Cristiana,,Pires', 'count': 1},
+                {'term': 'Enrique,,Sapena Ventura', 'count': 1},
+                {'term': 'Filipe,,Pereira', 'count': 1},
+                {'term': 'Fábio,,Rosa', 'count': 1},
+                {'term': 'Ida,,Zucchi', 'count': 1},
+                {'term': 'Ilia,,Kurochkin', 'count': 1},
+                {'term': 'Joshua D,,Ooi', 'count': 1},
+                {'term': 'Leilin,,Yu', 'count': 1},
+                {'term': 'Peng,,Jin', 'count': 1},
+                {'term': 'Peter J,,Eggenhuizen', 'count': 1},
+                {'term': 'Rong,,Tang', 'count': 1},
+                {'term': 'Ting,,Meng', 'count': 1},
+                {'term': 'Wannian,,Nie', 'count': 1},
+                {'term': 'Wei,,Lin', 'count': 1},
+                {'term': 'Xia,,Li', 'count': 1},
+                {'term': 'Xiang,,Ding', 'count': 1},
+                {'term': 'Xiangcheng,,Xiao', 'count': 1},
+                {'term': 'Yong,,Zhong', 'count': 1}
+            ],
```

Comment:
This is incredibly verbose and tedious. There is no point in listing dozens of terms if each term has cardinality of 1.

On top of that, I'm pretty sure you simply pasted this literal from an actual response, and maybe did some post-processing. So if there's a bug in the code affecting the facets, this literal will also reflect the bug.